app/templates/_build.gradle

```diff
@@ -1,14 +1,15 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 ext {
-    minSdkVersion = <%= androidMinSdkVersion %>
+    minSdkVersion = 19
```

Comment:
Yeah, for now. I will bring back on the next PR when I get the sign in stuff working properly